src/analysis/inner_product_space/orientation.lean

```diff
@@ -57,6 +57,17 @@ begin
   linarith,
 end
 
+/-- The change-of-basis matrix between two orthonormal bases with the opposite orientations has
+determinant -1. -/
+lemma det_to_matrix_orthonormal_basis_of_opposite_orientation
+  (h : e.to_basis.orientation ≠ f.to_basis.orientation) :
+  e.to_basis.det f = -1 :=
```

Comment:
I don't think `to_matrix` belongs in the names of lemmas that don't actually involve `to_matrix`; something more like `det_to_basis_of_opposite_orientation` would seem appropriate here. However, this isn't a request for a change to this PR, since several existing lemmas have the same issue (the only ones that actually reference `to_matrix`, and so where `to_matrix` seems appropriate in their names, are `orthonormal_basis.to_matrix_orthonormal_basis_mem_unitary` and `orthonormal_basis.to_matrix_orthonormal_basis_mem_orthogonal`).